tests/darkfi_oracle_check.py

```
@@ -0,0 +1,174 @@
+
+def _parse(text: str) -> dict[str, str]:
+    d = {}
+    for line in text.splitlines():
+        if "=" in line:
+            k, v = line.split("=", 1)
+            d[k.strip()] = v.strip()
+    return d
+
+
+def oracle_keys(oracle: str, mnemonic: str, account: int) -> dict[str, str]:
+    out = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)